.github/workflows/coverity.yml

```diff
@@ -0,0 +1,167 @@
+name: Coverity (Ubuntu 22.04, Python 3.11)
+on:
+  workflow_dispatch:
+  schedule:
+    # run daily at 00:00
+    - cron: '0 0 * * *'
+  pull_request:
+    paths:
+      - '.github/workflows/coverity.yml'
+
+permissions: read-all
+
+concurrency:
+  group: ${{ github.ref }}-tokenizers-cov-linux
+  cancel-in-progress: true
+
+env:
+  PYTHON_VERSION: '3.11'
+  OV_BRANCH: ${{ github.base_ref || github.event.merge_group.base_ref || github.ref }}
+
+jobs:
+  openvino_download:
+    name: Download prebuilt OpenVINO
+    outputs:
+      status: ${{ steps.openvino_download.outcome }}
+      ov_version: ${{ steps.openvino_download.outputs.ov_version }}
+      ov_wheel_source: ${{ steps.openvino_download.outputs.ov_wheel_source }}
+      docker_tag: ${{ steps.get_docker_tag.outputs.docker_tag }}
+    timeout-minutes: 10
+    defaults:
+      run:
+        shell: bash
+    runs-on: aks-linux-2-cores-8gb
+    container:
+      image: 'openvinogithubactions.azurecr.io/openvino_provider:0.1.0'
+      volumes: 
+        - /mount:/mount
+        - ${{ github.workspace }}:${{ github.workspace }}
+
+    steps:
+    - uses: openvinotoolkit/openvino/.github/actions/openvino_provider@master
+      id: openvino_download
+      with:
+        platform: ubuntu22
+        commit_packages_to_provide: wheels
+        revision: latest_available_commit
+
+    - name: Clone docker tag from OpenVINO repo
+      uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+      with:
+        repository: 'openvinotoolkit/openvino'
+        path: 'openvino'
+        ref: ${{ env.OV_BRANCH }}
+        sparse-checkout: |
+          .github/dockerfiles/docker_tag
+
+    - name: Save docker tag to output
+      id: get_docker_tag
+      run: |
+        docker_tag=$(cat openvino/.github/dockerfiles/docker_tag)
+        echo "docker_tag=$docker_tag" >> $GITHUB_OUTPUT
+
+  coverity_build:
+    name: Build for coverity
+    needs: [ openvino_download ]
+    timeout-minutes: 40
+    defaults:
+      run:
+        shell: bash
+    runs-on: aks-linux-8-cores-16gb
+    container:
+      image: openvinogithubactions.azurecr.io/ov_build/ubuntu_22_04_x64:${{ needs.openvino_download.outputs.docker_tag }}
+      volumes:
+        - /mount:/mount
+      options: -v ${{ github.workspace }}:${{ github.workspace }}
+    env:
+      CMAKE_GENERATOR: 'Ninja'
+      OPENVINO_REPO: ${{ github.workspace }}/openvino
+      INSTALL_DIR: ${{ github.workspace }}/openvino/install
+      OV_TOKENIZERS_INSTALL_DIR: ${{ github.workspace }}/openvino_tokenizers/install
+      BUILD_DIR: ${{ github.workspace }}/openvino_tokenizers/build
+      BUILD_TYPE: Release
+      BUILD_FAST_TOKENIZERS: 'ON'

```

Comment:
it is skipped in the build.sh later, I think we can explicitly set it in build.sh